e2e/fixtures/fdb_cluster.go

```diff
@@ -181,7 +181,8 @@ func MakeReconciliationOptionsStruct(
 	}
 
 	if reconciliationOptions.timeOutInSeconds == 0 {
-		reconciliationOptions.timeOutInSeconds = 4800
+		// Wait for 30 minutes as timeout.
+		reconciliationOptions.timeOutInSeconds = 1800
```

Comment:
Can we reduce this further? (If we don't have a reasonable guess as to how long the reconciliation would take, maybe we could log that information, and then change this based on that information, thanks!)

Reply:
Good call out, I open a new issue to see if we can reduce this further to e.g. ~15 minutes.